modApiServer/src/org/aion/api/server/rpc/ApiWeb3Aion.java

```diff
@@ -2421,7 +2421,7 @@ public RpcMsg ops_getTransactionReceiptListByBlockHash(Object _params) {
             b = blockCache.get(new ByteArrayWrapper(blockHash));
         } catch (Exception e) {
             // Catch errors if send an incorrect tx hash
-            return new RpcMsg(null, RpcError.INVALID_REQUEST, "Invalid Request" + e.getMessage());
+            return new RpcMsg(null, RpcError.INVALID_REQUEST, "Invalid Request" + e);
```

Comment:
let's update this to `"Invalid Request " + e +" " + e.getMessage()!=null? e.getMessage():''`

Reply:
updated.